### DISCUSSION TITLE
是否存在 docker compose 的构建

### DISCUSSION THREAD

**Discussion by caisxg** [tangly1024/NotionNext#2371] [General]
我不是很懂编程,求一份

**Comment by Daleveral** [comment]
![{B26EBB17-3E15-4319-9155-BA08D3CC79FE}](https://github.com/user-attachments/assets/55432cb4-78d9-45fb-8909-ef9b71171467)
点击右侧 packages 按钮能发现打包好的 docker 镜像, 没有的话点击右上角设置按钮勾选上 packages
